hoomd/md/methods.py

```diff
@@ -1137,11 +1158,13 @@ class Brownian(MethodRATTLE):
             coefficient can be directly set instead of the ratio of particle
             diameter (:math:`\gamma = \alpha d_i`). The type of ``gamma``
             parameter is either positive float or zero.
+            :math:`[\mathrm{mass} \cdot \mathrm{time}^{-1}]`
 
         gamma_r (TypeParameter[``particle type``, [`float`, `float`, `float`]]):
             The rotational drag coefficient can be set. The type of ``gamma_r``
             parameter is a tuple of three float. The type of each element of
             tuple is either positive float or zero.
+            :math:`[\mathrm{mass} \cdot \mathrm{time}^{-1}]`
```

Comment:
`gamma_r` multiplies an angular velocity which results in a torque. Therefore, its units should be force * distance * radians^{-1} * time^{-1}.